parse_grid.py

```
@@ -163,9 +163,18 @@ def aggregate_appliance_availability(
     return list(appliance_dict.values())
 
 
+def _get_soup(html_content: str) -> BeautifulSoup:
+    """Get BeautifulSoup object with lxml fallback."""
+    try:
+        return BeautifulSoup(html_content, "lxml")
+    except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)